libindy_vdr/Cargo.toml

```diff
@@ -60,6 +60,8 @@ thiserror = "1.0"
 time = { version = "=0.3.20", features = ["parsing"] }
 url = "2.2.2"
 zmq = "0.9"
+tokio = { version = "1.35.1", features = ["full"] }
```

Comment:
async-lock would be a much lighter dependency

Reply:
Done